pym/gentoolkit/revdep_rebuild/settings.py

```diff
@@ -121,7 +121,7 @@ def parse_options():
 	if args.library:
 		settings['library'].update(set(args.library))
 	settings['USE_TMP_FILES'] = not args.ignore
-	settings['pass_through_options'] = " " + " ".join(args.portage_options)
+	settings['pass_through_options'].extend(args.portage_options)
```

Comment:
This actually modifies the list in DEFAULTS as well, due to the shallow copy here:
```python
settings = DEFAULTS.copy()
```
Therefore, we should do a deep copy:
```python
settings = copy.deepcopy(DEFAULTS)
```

Reply:
Thanks for noticing.
However, deepcopy makes weird stuff with stdin/stderr/stdout. Looks like file handles can't be deepcopied...

I made a new commit to only copy 'pass_through_options'' and 'library'.

Reply:
Good, thank you. Please squash the commits together, and also please add a `Signed-off-by` line to the commit message as described here:

https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin